freqtrade/rpc/telegram.py

```diff
@@ -610,6 +616,25 @@ def _forcesell(self, update: Update, context: CallbackContext) -> None:
         except RPCException as e:
             self._send_msg(str(e))
 
+    def _forcebuy_action(self, pair, price=None):
+        try:
+            self._rpc._rpc_forcebuy(pair, price)
+        except RPCException as e:
+            self._send_msg(str(e))
+
+    def _forcebuy_inline(self, update: Update, _: CallbackContext) -> None:
```

Comment:
I'm not entirely sure about this (i never used inline-keyboards with telegram so far) - but would we need to add the `@authorize_only` tag to this method too, or is this only reachable as part of an answer?

Reply:
Let me double-check that...

Reply:
Trying this, i found no way to get to this callback without having an authorization first - so all fine.